src/include/sof/lib_manager.h

```diff
@@ -0,0 +1,204 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Jaroslaw Stelter <jaroslaw.stelter@intel.com>
+ *         Pawel Dobrowolski <pawelx.dobrowolski@intel.com>
+ */
+
+/*
+ * Library manager implementation for SOF is using codec adapter API for
+ * loadable native and external libraries. Depends of information received from
```

Comment:
As already pointed out, there is no longer 'codec adapter' but we renamed this to 'module adapter' please fix the doc